README.md

```diff
@@ -37,86 +37,172 @@ Note that in the future, we plan to scan the source repository for a thumbnail i
 Follow these guidelines when creating your thumbnail to help us maintain a beautiful, consistent, and streamlined catalog.
 They are listed roughly in order of most strict to least strict.
 
-- **1000 x 1000 pixels**
+
+<details>
+<summary>
+1000 x 1000 pixels
+</summary>
```

Comment:
I think you may need a blank line after this `</summary>` for GitHub to parse the contents as markdown